lib/igniter/util/install.ex

```diff
@@ -1,5 +1,7 @@
 defmodule Igniter.Util.Install do
-  @moduledoc false
+  @moduledoc """
+  Tools for installing packages and running their associated Igniter installers, if present.
```

Comment:
Lets add a line here about how this is not composable like other installers, and this is meant to be used if you are building custom tooling on top of igniter aside from "standard" generators/installers/patchers. Feel free to link to your project as an example.